fix_encode.sh

```
@@ -0,0 +1,25 @@
+# Fix WEB-INF/jsp/messenger/ViewAttachment.jsp
+sed -i 's/<%@ page import="org.owasp.encoder.Encode" %>/<%@ page import="io.github.carlos_emr.carlos.utility.SafeEncode" %>/g' src/main/webapp/WEB-INF/jsp/messenger/ViewAttachment.jsp
+sed -i 's/Encode.for/SafeEncode.for/g' src/main/webapp/WEB-INF/jsp/messenger/ViewAttachment.jsp
+
+# Fix WEB-INF/jsp/messenger/ViewPDFAttachment.jsp
+sed -i 's/<%@ page import="org.owasp.encoder.Encode" %>/<%@ page import="io.github.carlos_emr.carlos.utility.SafeEncode" %>/g' src/main/webapp/WEB-INF/jsp/messenger/ViewPDFAttachment.jsp
+sed -i 's/Encode.for/SafeEncode.for/g' src/main/webapp/WEB-INF/jsp/messenger/ViewPDFAttachment.jsp
+
+# Fix WEB-INF/jsp/messenger/attachmentFrameset.jsp
+sed -i 's/<%@ page import="org.owasp.encoder.Encode" %>/<%@ page import="io.github.carlos_emr.carlos.utility.SafeEncode" %>/g' src/main/webapp/WEB-INF/jsp/messenger/attachmentFrameset.jsp
+sed -i 's/<%@ taglib uri="https:\/\/www.owasp.org\/index.php\/OWASP_Java_Encoder_Project" prefix="e" %>/<%@ taglib uri="\/WEB-INF\/carlos.tld" prefix="carlos" %>/g' src/main/webapp/WEB-INF/jsp/messenger/attachmentFrameset.jsp
+sed -i 's/${e:for/${carlos:for/g' src/main/webapp/WEB-INF/jsp/messenger/attachmentFrameset.jsp
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)